subsys/Makefile

```diff
@@ -8,3 +8,4 @@ obj-$(CONFIG_DISK_ACCESS) += disk/
 obj-$(CONFIG_CPLUSPLUS) += cpp/
 obj-y += logging/
 obj-y += debug/
+obj-$(CONFIG_MCUBOOT_IMG_MANAGER) += dfu/
```

Comment:
why are we making a subsys like dfu depend on a signle bootloader implementation like MCUBOOT? We need to abstract this and have the MCU_IMG_MANAGER be an implementation, i.e. one out of possible many.